### ISSUE TITLE
Bug: Problem with 3.5 system

### ISSUE THREAD

**Issue by irbian** [otigon/automated-jb2a-animations#70]
I´m having this bug now when toggling some options on the sheet:

```
TypeError: Cannot read property 'toLowerCase' of undefined
    at Function.bind (item-sheet-config.js:32)
    at autoAnimations.js:246
    at Function._call (eval at <anonymous> (libWrapper-errors.js:249), <anonymous>:4:14)
    at Function.call (foundry.js:179)
    at ItemSheetPF._render (foundry.js:2096)
    at async ItemSheetPF._render (foundry.js:2728)
```

**Comment by otigon** [comment]
This  issue should be resolved tomorrow when the new 3.5 version is released. I'll check back in to make sure

**Comment by otigon** [comment]
Issue should now be resolved with the latest version of A-A
